### PR TITLE
Linking criteria and adding file and env variables access

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What is this feature?**

Extending plugin signing and publishing criteria and linking the criteria in the publishing documentation.

### CODE REVIEW

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

**Review by wbrowne — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM

File: docs/sources/developers/plugins/publish-a-plugin.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Before you submit your plugin, we ask that you read our guidelines and frequentl
 
 To speed up the time it takes to review your plugin:
 
+- Get familiar with our pluign [publishing and signing criteria](publishing-and-signing-criteria.md)
```
Comment (marefr) [inline docs/sources/developers/plugins/publish-a-plugin.md:89]:
```suggestion
- Get familiar with our plugin [publishing and signing criteria](publishing-and-signing-criteria.md)
```

**Review by mckn — APPROVED** [review]
LGTM, apart from the small not @maref already added a suggestion for.

**Pull request merged**